lib/main.dart

```diff
@@ -1,18 +1,110 @@
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const CloneInstagramApp());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+class CloneInstagramApp extends StatelessWidget {
```

Comment:
Should write in a separate file such as: main_app.dart

Reply:
I changed